src/Context/Context.php

```diff
@@ -74,7 +74,7 @@ public function isClosed(): bool {
 	}
 
 	public static function getTime(): int {
-		return (int) microtime(true) * 1000;
+		return (int) (microtime(true) * 1000);
```

Comment:
Are those ints 64-bit? :)

Reply:
Yes, as long as it runs on 64 bit :)

There are extensions that support distinct bit widths, but PHP core only has this. We could check for 32 bit processors and perhaps set this value as a string, but I'm not sure if the API accepts it? I'll happily work that out if that's the case.

Reply:
Also, we can define a clause in the `composer.json` file to require a 64-bit processor. Composer will refuse to install the SDK on 32 bit processors.

Reply:
#2 proposes this.